application/models/TemplateConfiguration.php

```diff
@@ -783,9 +784,9 @@ class="btn btn-primary btn-block selector--ConfirmModal">
             href="' . $sUninstallUrl . '"
             data-post=\'{ "templatename": "' . $this->template_name . '" }\'
             data-text="' . gT('This will reset all the specific configurations of this theme.') . '<br>' . gT('Do you want to continue?') . '"
-            data-button-no="' . gt('Cancel') . '" 
```

Comment:
For future, can you fix your editor to not apply whitespace changes automatically, please? It makes code review harder. :(

Reply:
Yes, sorry …